CRM/Core/DAO.php

```diff
@@ -652,9 +652,36 @@ public function save($hook = TRUE) {
         $preEvent->eventID = $eventID;
         \Civi::dispatcher()->dispatch("civi.dao.preInsert", $preEvent);
       }
+      // If we are in MariaDB disable Strict Trans Table SQL mode to prevent hard fail in multilingual mode
+      $version = self::getDatabaseVersion();
+      if (stripos($version, 'mariadb') !== FALSE && CRM_Core_I18n::isMultiLingual()) {
+        CRM_Utils_SQL::disableStrictTransTable();
```

Comment:
Just noting that you are not re-enabling it - but I think that's OK in that it is for the session only & think that would not cause any inconsistencies if it lasted until them (fullGroupBy I think might be more important to re-enable so that any FGB bugs are not intermittant)

Reply:
It looks like it gets re-enabled lower down line 685 (in fact, unconditionally on whether you even had it set before, so might inadvertently affect subsequent write queries that don't come thru save()).

Overall, personally I'm undecided on whether making mariadb act more like mysql is good or not.